src/containers/tab-sidemenu/goals-menu.tsx

```
@@ -0,0 +1,24 @@
+import Goals from "@/assets/goals.svg";
+import { IsAddingProps } from "@/types/tab-sidemenu";
+
+import CustomButton from "./custom-button";
+import Menus from "./menus";
```

ESLint

[error] Missing file extension for "./menus"

(import/extensions)